src/nvim/getchar.c

```diff
@@ -4041,20 +4012,17 @@ char_u *vim_strsave_escape_csi(char_u *p)
   return res;
 }
 
-/*
- * Remove escaping from CSI and K_SPECIAL characters.  Reverse of
- * vim_strsave_escape_csi().  Works in-place.
- */
-void vim_unescape_csi(char_u *p)
+/// Remove escaping from CSI and K_SPECIAL characters.  Reverse of
+/// vim_strsave_escape_ks().  Works in-place.
+void vim_unescape_ks_csi(char_u *p)
 {
   char_u *s = p, *d = p;
 
   while (*s != NUL) {
     if (s[0] == K_SPECIAL && s[1] == KS_SPECIAL && s[2] == KE_FILLER) {
       *d++ = K_SPECIAL;
       s += 3;
-    } else if ((s[0] == K_SPECIAL || s[0] == CSI)
```

Comment:
Keeping this should be harmless, as `CSI KS_EXTRA KE_CSI` is not valid UTF-8 anyway. But since other similar unescaping code has already been removed from other functions like `vgetc()` (with the removal of `FEAT_GUI`), I'll remove this `s[0] == CSI` as well.